env/production/replica_count.yaml

```diff
@@ -72,4 +72,4 @@ metadata:
   namespace: notification-canada-ca
 spec:
   minReplicas: 1
-  maxReplicas: 1
+  maxReplicas: 2
```

Comment:
We probably don't need to ever have more than one `document-download-api` pod, but it doesn't hurt to have another available.

Reply:
I increased it as I saw some errors in the past where the cause might have been insufficient resources. But I didn't investigate long enough to be certain of it.